BookingsApi/BookingsApi.DAL/Queries/GetVhoNonAvailableWorkHoursQuery.cs

```diff
@@ -29,8 +29,10 @@ public async Task<List<VhoNonAvailability>> Handle(GetVhoNonAvailableWorkHoursQu
             
             if(justiceUser == null)    
                 return null;
+
+            var hours = justiceUser.VhoNonAvailability.Where(x => x.Deleted != true).ToList();
```

Comment:
what happens if `justiceUser.VhoNonAvailability` is null? Might be better to preserve the original null check and do something like:

`return justiceUser.VhoNonAvailability?.Where(x => x.Deleted != true).ToList() ?? new List<VhoNonAvailability>();`

Reply:
changed